features/step_definitions/user_steps.rb

```diff
@@ -119,6 +119,7 @@
 ### THEN ###
 Then /^I should be signed in$/ do
```

Comment:
could this step name tell us more accurately about what it is doing under the hood?